frontend/src/components/commons/ConfirmStartTest.jsx

```diff
@@ -11,6 +11,7 @@ class ConfirmStartTest extends Component {
   };
 
   render() {
+    //TODO: replace `timeUnlimited` with dynamic real time value to complete the test.
```

Comment:
I'm fine with this being another PR, but as an FYi, we currently do get the time limit from the test-meta-data endpoint:
![image](https://user-images.githubusercontent.com/2746350/60334780-441f5d00-996a-11e9-82a4-b7481d268e70.png)
Currently, both tests (pizza and sample) are set to null, since we had not defined a time limit for either.

Reply:
this is more a note for the PR that does add the actual time limit in

Reply:
Good to know. I think it'd be helpful to explicitly load those values into redux, or define a shape somewhere because it was hard for me to tell what values were already there.